src/lavinmq/http/controller.cr

```diff
@@ -191,38 +191,35 @@ module LavinMQ
       end
 
       private def refuse_unless_vhost_access(context, user, vhost)
+        return if user.tags.any? &.administrator?
```

Comment:
Grant access to vhost if administrator (not necessary have vhost permission). Otherwise check if user have vhost permission set regardless if it has tags or not. 

Have used it in combination with the other tag checks. E.g. refuse_unless_managment.